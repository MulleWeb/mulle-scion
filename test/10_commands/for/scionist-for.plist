<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN"
                       "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>Club</key>
	<array>
		<dict>
			<key>name</key>
			<string>VfL Bochum 1848</string>
		</dict>
	</array>
	<key>Liga</key>
	<array>
		<dict>
			<key>clubs</key>
			<array>
				<dict>
					<key>name</key>
					<string>VfL Bochum 1848</string>
				</dict>
			</array>
			<key>name</key>
			<string>2. Liga</string>
		</dict>
		<dict>
			<key>name</key>
			<string>Bundesliga</string>
			<key>clubs</key>
			<array/>
		</dict>
	</array>
</dict>
</plist>