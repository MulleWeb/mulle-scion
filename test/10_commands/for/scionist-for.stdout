Bundesliga
2. Liga
